docs/api-docs/customers/customers-subscribers-overview.md

```diff
@@ -208,6 +205,10 @@ When creating a customer there are two ways to set customers passwords on the V3
 
 [Password confirmation](https://developer.bigcommerce.com/api-reference/customer-subscribers/customers-api/customers/createanewcustomer) and [validation](https://developer.bigcommerce.com/api-reference/customer-subscribers/customers-api/customer-passwords/validatecustomerpassword) are still available under V2 customers. 
 
+## Subscribers API

```

Comment:
> The Subscribers API allows you to manage subscribers who have signed up for the store’s newsletter.

[the Storefront Email Subscriptions API](https://developer.bigcommerce.com/api-reference/storefront/storefront-subscriptions/subscription/postsubscriptions) is for creating or updating customer marketing email subscriptions. 

[Subscribers V3](https://developer.bigcommerce.com/api-reference/store-management/subscribers/subscribers/getsubscribers) is for managing subscriber customer accounts.